taskfiles/k8s.yml

```diff
@@ -239,3 +239,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - mkdir helm-operator-release
+      - defer 'rm -r helm-operator-release'
+      - yq -i '.name = "redpanda-operator-nightly"' charts/operator/Chart.yaml
```

Comment:
Please add some comments as to why these steps are present.

Reply:
ok